rfq/manager.go

```diff
@@ -199,9 +204,10 @@ func (m *Manager) startSubsystems(ctx context.Context) error {
 	// Initialise and start the quote negotiator.
 	m.negotiator, err = NewNegotiator(
 		NegotiatorCfg{
-			PriceOracle:      m.cfg.PriceOracle,
-			OutgoingMessages: m.outgoingMessages,
-			ErrChan:          m.subsystemErrChan,
+			PriceOracle:               m.cfg.PriceOracle,
+			OutgoingMessages:          m.outgoingMessages,
+			SkipAcceptQuotePriceCheck: m.cfg.SkipAcceptQuotePriceCheck, // nolint: lll
```

Comment:
nit: use nolint directive above this line instead of at the end? Below as well.

Reply:
I'm starting to see a pattern here. Never at the end right?